src/support/z_access_syslog.erl

```diff
@@ -53,6 +53,7 @@ start_link(Ident, Opts, Facility, Level) ->
     z_buffered_worker:start_link(?MODULE, ?MODULE, [[Ident, Opts, Facility], Level]).
 
 log_access(_LogData) ->
+    %% TODO Do access logging here?
```

Comment:
@mmzeeman This is probably the place you’re referring to for the buffered syslogging.